pkgs/development/tools/analysis/tfsec/default.nix

```
@@ -2,23 +2,23 @@
 
 buildGoPackage rec {
   pname = "tfsec";
-  version = "0.27.0";
+  version = "0.36.10";
 
   src = fetchFromGitHub {
-    owner = "liamg";
+    owner = "tfsec";
     repo = pname;
     rev = "v${version}";
-    sha256 = "12n6l18g0kg28clzyzwc2lq9rnch3vshp07isgvfp7193bhf8hyv";
+    sha256 = "11kv13d4cw515r79azfha1ksmvsha1rvg0jak9nvz9ggivyn0s7a";
   };
 
-  goPackagePath = "github.com/liamg/tfsec";
+  goPackagePath = "github.com/tfsec/tfsec";
 
   buildFlagsArray = [ "-ldflags=-s -w -X ${goPackagePath}/version.Version=${version}" ];
 
   meta = with lib; {
-    homepage = "https://github.com/liamg/tfsec";
+    homepage = "https://github.com/tfsec/tfsec";
     description = "Static analysis powered security scanner for your terraform code";
     license = licenses.mit;
-    maintainers = [ maintainers.marsam ];
+    maintainers = [ maintainers.anhdle14 ];
```

```suggestion
    maintainers = with maintainers; [ marsam anhdle14 ];
```
Why did you remove @marsam? Usually I only remove maintainers with their consent or if the package has been unmaintained for YEARS.

---

Sorry, I will exclude my name from it right now, was thinking that it is for the latest maintainers

---

No, you're fine to add your name. Many packages in nixpkgs have multiple maintainers. Maintaining mostly is just a loose commitment to try to keep the package up to date, and be notified for testing purposes via GitHub when someone submits a PR for this package (e.g. ``marsam`` should've been notified for this package originally)

The syntax would be as I showed above.